assignment2.txt

```diff
@@ -0,0 +1,29 @@
+Emma Ladouceur homework2 questions
+
+Question 1
+	Absolutely! Its quite common to mix two operators in equations. For example for the quadratic equation divition multiplication and square roots are all used. The important thing is remebering order of operations and parenthesis
+
+
+Question 2
+	Systax errors involve forgetting a ; or making a mistake in the coding of a program whereas logical errors are issues with the program and how it goes about completing its task.
+
+
+Question 3
+	True && x< 25 && x!=12
+	True && False && True
+	False && True
+	False (Which i believe in c is 0?)
```

Comment:
Hm, not quite! The answer is actually true, which is 1 in C. Go over again and find where you went wrong!